state/memcached/memcached.go

```diff
@@ -125,6 +130,10 @@ func (m *Memcached) parseTTL(req *state.SetRequest) (*int32, error) {
 		}
 		parsedInt := int32(parsedVal)
 
+		if parsedInt < 0 {
```

Comment:
In general negative TTL means to persist the item in Dapr State Store APIs. Can we change this to not through an error but persist the item instead?

Reply:
Since it seems `0` means not expiring, just remap `<1` to `0`

Reply:
Sure. I will replace a -1 with 0 (as that is the value for no expiration on memcached) and error on any other negative number.

Reply:
I will keep 0 as a pass through. I guess if you are a memcached user you expect 0 to means "persist" and be equivalent to Dapr's -1